extensions/scientific/examples/example-merraclim.json

```diff
@@ -1,5 +1,9 @@
 {
   "stac_version": "0.7.0",
+  "stac_extensions": [
```

Comment:
```
"stac_extensions": {
        "eo": "0.0.1",
        "scientific": "0.0.1"
    }
```

Reply:
See my comment below. I'm pretty sure we settled on arrays as the version information is basically included in the URL as the referenced schema is versioned.